source/components/Options.js

```diff
@@ -12,6 +12,10 @@ import type {
   ElementRef
 } from 'react';
 
+// external libraries
+import { isFunction, get } from 'lodash';
+import createRef from 'create-react-ref/lib/createRef';
```

Comment:
I think we can remove that library since we are using `"react": ">=16.8.6"` and just use `React.createRef`

Reply:
✅